src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -317,35 +322,37 @@ class InferenceProfiler {
   /// measures (we can't get the exact server status right before the first
   /// request and right after the last request in the measurement window).
   /// \param concurrent_request_count The concurrency level for the measurement.
-  /// \param summary Appends the measurements summary at the end of this list.
-  /// \param meets_threshold Returns whether the setting meets the threshold.
-  /// \param is_stable Returns whether the measurement is stable.
+  /// \param perf_status_vec Appends the measurements summary at the end of this
+  /// list. \param meets_threshold Returns whether the setting meets the
```

Comment:
the \param should be at the start of the line. vscode auto-format doesn't always honor that.

Reply:
nice catch. Updated.